README.md

```diff
@@ -54,6 +54,18 @@ It can be generated by running command `make generate-openapi-test`
 Generated spec can be found under 
 `build/rest/openapi/spec`
 
+##### Verifiable Credential Test Suite
+In order to check compatibility of Verifiable Credential implementation in aries-framework-go with 
```

Comment:
To test compatibility of the verifiable credential packages with ...

Reply:
Thanks! I will fix that.

Reply:
Fixed